my_enumerable.rb

```diff
@@ -0,0 +1,13 @@
+module Enumerable
+  def all(list)
+    list.all? { |num| num < 5 }
+  end
+
+  def any(list)
+    list.any? { |num| num == 5 }
+  end
+
+  def filter(list)
```

Comment:
`filter` should return an array containing all elements of the enum for which the given block returns a true value. For example if the list = [1, 2, 3, 4], then `list.filter {|e| e.even?}` should return `[2, 4]`.